src/EventStore.Core/Services/Gossip/GossipServiceBase.cs

```diff
@@ -168,10 +168,18 @@ public abstract class GossipServiceBase : IHandle<SystemMessage.SystemInit>,
 
 			message.Envelope.ReplyWith(new GossipMessage.SendGossip(_cluster, _memberInfo.HttpEndPoint));
 
-			if (_cluster.HasChangedSince(oldCluster))
+			if (_cluster.HasChangedSince(oldCluster)) {
 				LogClusterChange(oldCluster, _cluster, $"gossip received from [{message.Server}]");
+				PublishIfClusterHasMultipleVersions(_cluster);
```

Comment:
I think that this call should be moved out of the "cluster has changed" if check. The different versions warning should be published whenever a gossip comes back with a different version, even if the cluster has not changed.

Reply:
Would "spamming" the log be okay? Unless something is changed in cluster, logging again and again doesn't give any extra information.

Reply:
Think spamming in this case is what we want to "encourage" people to upgrade 🤔 unless it turns into flooding 😅